gef.py

```diff
@@ -2788,6 +2974,30 @@ def get_os():
     return platform.system().lower()
 
 
+@lru_cache()
+def is_qemu():
+    if not is_remote_debug():
+        return False
+    response = gdb.execute('maintenance packet Qqemu.sstepbits', to_string=True, from_tty=False)
```

Comment:
is there any way this GDB command raises an exception that needs to be handled?

same question also for L2989 and L2997

Reply:
Don't think so